Templates/ManualTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "defaultFunctionName": "ManualTriggerCSharp",
+    "description": "A C# function without triggers, inputs, or outputs",
+    "name": "ManualTrigger - C#",
+    "trigger": "MaunualTrigger",
+    "language": "C#",
+    "category": [ "Experimental" ]
```

Comment:
Both of these templates should be moved to the "Core" category, peer to the Empty template which is also Core